vivy-idea-plugin/src/main/resources/META-INF/file/checkstyle.xml

```diff
@@ -72,7 +75,6 @@
         </module>
         <module name="RightCurly">
             <property name="id" value="RightCurlyAlone"/>
-            <property name="shouldStartLine" value="true"/>
```

Comment:
what does it do?

Reply:
I deleted it because it was deleted from checkstyle at all. It was checking single-line if statements, which are covered by option in our case.  https://github.com/checkstyle/checkstyle/issues/4022